app/routes/api/stage-execution.js

```diff
@@ -98,7 +98,8 @@ export const stageExecutionHandlers = [{
           // note that even though it using request.payload.applicationReference
           // it can actually be the claim reference which is passed in the request
           // see AHWR-454 for further details
-          await updateClaimByReference({ reference: request.payload.applicationReference, statusId, updatedBy: request.payload.executedBy, sbi })
+          const note = null
```

Comment:
I understand what's happening here, this route does not add a note as it's the stage executions we're trying to remove, but given that const appears directly under an unrelated comment, would it be an idea to make it clear what's happening here to avoid confusion when someone later looks at this? Unless you're planning to rip this stage execution processing out very soon as part of the wider changes going on

Reply:
I am planning to remove the stage executions soon as we wont be using them. But I can move the declaration of note above the comments until then.